editor/components/block-list/style.scss

```diff
@@ -1,4 +1,4 @@
-.editor-block-list__block {
+.edit-post-visual-editor .editor-block-list__block {
```

Comment:
Sorry for all these leftover classnames, I was pretty sure, I changed everything on the PR :)